### DISCUSSION TITLE
v6: how to monitor a specific port?

### DISCUSSION THREAD

**Discussion by czxin788** [ccfos/nightingale#1715] [Q&A] [answered]
请问v6版本，怎么监控指定端口

**Comment by UlricQin** [comment, marked as answer]
https://flashcat.cloud/docs/content/flashcat-monitor/nightingale-v6/faq/n9e-can-monitor-xx/

可以先看看上面的文档。至于采集端口存活，手段就很多了，比如使用 telegraf 或者 categraf 的 net_response 插件
